pkg/cli/options_test.go

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cli_test
+
+import (
+	"github.com/projectriff/riff/pkg/cli"
+	"github.com/projectriff/riff/pkg/testing"
+)
+
+func TestListOptions(t *testing.T) {
```

Comment:
Yeah, this wasted several minutes of my time. I eventually turned off the inspection, but it would be better not to have to.

Reply:
I'm not familiar with GoLand, what is the behavior that's problematic?

The reason I added the alias is because without it you have two different "testing" packages imported. I tried to put everything most tests will need into a single package.

Reply:
By default GoLand flags test methods when it thinks they have invalid signatures. It's questionable whether this signature is valid or invalid, although it happens to work.

Given the prevalence of the standard `testing` package, I think be worth adding a `rifftest` or `rifftesting` package distinct from `testing` and using the standard `testing` package in test signatures. This would reduce the surprise factor and keep GoLand happy.

Reply:
ok, I'll split the packages

Reply:
done